sematic/ui/packages/main/config-overrides.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+
+module.exports = function override(config, env) {
+    //do stuff with the webpack config...
+    // console.log({config: JSON.stringify(config, null, 4),});
```

Comment:
Why is this commented out?

Reply:
Good catch. This shouldn't be there. Removed. Thanks!